crates/config/src/operator_registration.rs

```diff
@@ -0,0 +1,12 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Default, Serialize, Deserialize, PartialEq, Eq, Clone)]
+pub struct OperatorRegistrationConfig {
+    pub register_operator: bool,
+    pub operator_to_avs_registration_sig_salt: String,
```

Comment:
i think we can name this AvsOperatorRegistrationConfig and add another struct for EigenlayerOperatorRegistrationConfig so people are not confused.

we can make the params optional that are only available in pre slashing or post slashing. like we did in writer structs.

Ex : allocation_delay, salt etc 
can make a PR if you agree with this